python_modules/dagster/dagster/_core/definitions/asset_graph.py

```diff
@@ -77,7 +78,10 @@ def metadata(self) -> ArbitraryMetadataMapping:
 
     @property
     def tags(self) -> Mapping[str, str]:
-        return self._spec.tags
+        return {
```

Comment:
Does this mean that `asset_graph.get(asset_key).tags` will return a different value than `asset_graph.get(asset_key).to_asset_spec().tags`? Is that desirable?

Reply:
Moved the combine step to the external asset node, so asset node will still retain kinds as a separate notion.